test/draw_test.py

```diff
@@ -4242,7 +4242,7 @@ def test_circle__kwarg_invalid_types(self):
                        {'surface' : surface,
                         'color'   : color,
                         'center'  : center,
-                        'radius'  : 1.1,  # Invalid radius.
+                        'radius'  : "1",  # Invalid radius.
```

Comment:
Since the radius parameter can now be a float it would be good to add in a test or extend some current tests to ensure float values are accepted.

One test to change could be `test_circle__valid_radius_values()`. e.g. change `for radius in (-10, -1, 0, 1, 10):` to `for radius in (-10, -1, -1.1, 0, 1, 1.1, 10):`. That test is marked as `@unittest.expectedFailure` due to another issue so it currently won't detect any new problems right now. Another test to alter could be `test_circle__floats()`. e.g. change the radius values to floats.

Reply:
I have altered `test_circle__floats()` in new commits